benchmarks/e2e/tests/block_multitenant_resources/block_multitenant_resources.go

```diff
@@ -0,0 +1,114 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	configutil "github.com/realshuting/multi-tenancy/benchmarks/e2e/config"
```

Comment:
@phoenixking25 - this should refer to https://github.com/kubernetes-sigs/multi-tenancy/blob/master/benchmarks/e2e/config/config.go

Reply:
https://github.com/kubernetes-sigs/multi-tenancy/pull/538
@JimBugwadia sure, to run the tests i had to put the path like this before the given merge, i was just waiting for that merge so my pr dont create conflict